doc/source/conf.py

```diff
@@ -40,7 +40,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["sphinx.ext.autodoc", "sphinx.ext.doctest", "sphinx.ext.intersphinx"]
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.doctest", "sphinx.ext.intersphinx", 'sphinx.ext.napoleon']
```

Comment:
`napoleon` is a parser that allows to generate docs from Google's style docstrings.